Dockerfile-neon

```diff
@@ -55,6 +55,10 @@ RUN sudo ln -s /home/kdedev/.kdesrc-buildrc /root/.kdesrc-buildrc && \
     sudo ln -s /home/kdedev/kdesrc-build /root/kdesrc-build
 # setup kdedev account
 RUN sudo apt-get install -y sudo && sudo bash -c "echo 'kdedev ALL=NOPASSWD: ALL' >>  /etc/sudoers"
```

Comment:
You don't need to use `sudo` inside a Docker environment, the USER is root by default.
`user kdedev` is just below

Reply:
Unless the baseimage runs already with a specific USER ?

Reply:
Yes, the base image starts with the "neon" user.